server/platform/shared/filestore/localstore.go

```diff
@@ -211,7 +211,6 @@ func appendRecursively(basePath, path string, maxDepth int) ([]string, error) {
 		}
 		if dirEntry.IsDir() {
 			if maxDepth <= 0 {
-				mlog.Warn("Max Depth reached", mlog.String("path", entryPath))
```

Comment:
But the administrator should be aware that some files/directories will get skipped. This is important when running compliance export for example.

Maybe we can have a better error message: "Max depth reached, skipping any further directories under:"

Reply:
Hmm, but it will always print warning if you want to just list directories with 0 recursion.